pages/docs/tooling/tooling-builtin.md

```diff
@@ -10,10 +10,10 @@ It provides an easy-to-use interface to tooling API of the preCICE library.
 
```

Comment:
As part of this PR, maybe also extend the introduction a bit. "an easy-to-use interface to tooling API" is a bit vague for the user.

Maybe write something like:

```suggestion

With `binprecice`, you can get the installed preCICE version, generate a reference of all available configuration options, as well as check your configuration file for basic configuration issues.
```

Reply:
Good point